### PR TITLE
Added basic Telnet scanner - #27

### PR DESCRIPTION
**Pull request by decidedlygray**
This PR adds basic Telnet scanner functionality. It also adds Mirai botnet credentials as the default scan creds for telnet.

Telnetlib is not ideal for scanning, but with a little hackery it becomes a "good enough" solution for up to a few hundred hosts.

### CODE REVIEW

**Comment by ztgrace** [comment]
This is awesome! I'll do a little testing merge it tonight.

File: changeme/scanners/telnet.py [hunk 1]
```diff
@@ -1,29 +1,78 @@
...
     def _check(self):
         try:
-            telnet = telnetlib.Telnet(str(self.target), int(self.port), int(self.config.timeout))
-            telnet.read_until("login: ")
+            telnet = telnetlib.Telnet(str(self.target.host))
+            timeout_allowed = int(self.cred['auth']['blockingio_timeout'])
+            wait_for_pass_prompt = int(self.cred['auth']['telnet_read_timeout'])
+
+            retval = telnet.open(str(self.target.host), int(self.target.port), timeout=timeout_allowed)
+            retval._has_poll = False    # telnetlib hackery :)
```
Comment (ztgrace) [inline changeme/scanners/telnet.py]: What's this line do? I keep getting an error:

    [21:43:45][telnet][_check] Error: 'NoneType' object has no attribute '_has_poll'

**Comment by decidedlygray** [comment]
If you look into the telnetlib code, under some condition it gets put into a mode where it polls for results every so often to see if it should exit/stop blocking, and that was to prevent it from going into that mode.

Try removing it.

The reason I suggest that: it worked on Kali without that line. I moved to a clean Ubuntu environment before committing to test it further and ended up needing to add that line to prevent a complete lockup after 30 or so threasds were spun up

Another option would be to add an if statement to handle the condition where it’s None, but maybe we don’t need it at all

**Comment by ztgrace** [comment]
I'm pulling the changes into the development branch to clean them up.

**Pull request merged**